packages/terra-navigation-side-menu/src/MenuItem.scss

```diff
@@ -0,0 +1,47 @@
+
+:local {
+  .menu-item {
+    align-items: stretch;
+    color: var(--terra-navigation-side-menu-item-color, #1c1f21);
+    display: flex;
+    font-size: var(--terra-navigation-side-menu-item-size, 1.071rem);
+    padding-bottom: var(--terra-navigation-side-menu-item-padding-bottom, 0.8225rem);
+    padding-left: var(--terra-navigation-side-menu-item-padding-left, 1.2143rem);
+    padding-right: var(--terra-navigation-side-menu-item-padding-right, 0.7143rem);
+    padding-top: var(--terra-navigation-side-menu-item-padding-top, 0.8225rem);
+    text-decoration: none;
+    width: 100%;
+
+    &:hover,
+    &.is-hovered {
+      background: var(--terra-navigation-side-menu-item-background-hover, linear-gradient(-90deg, #fff, #f1f1f2));
```

Comment:
Should these be white or transparent? I'm curious because we recently had this PR #79

Reply:
Unfortunately I had to revert the change, as safari doesn't properly handle linear-gradient with transparent.